A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.html

```diff
@@ -0,0 +1,164 @@
+        </g>
+
+        <!--APU-->
+        <g id="APU">
+            <text x="165" y="250" text-anchor="end" alignment-baseline="central">APU</text>
+            <line x1="182.5" y1="242.5" x2="182.5" y2="257.5"></line>
+            <line x1="182.5" y1="242.5" x2="167.5" y2="250"></line>
+            <line x1="182.5" y1="257.5" x2="167.5" y2="250"></line>
+        </g>
+
+        <!--Left-->
+        <g id="leftPipes">
+            <line class="FlowShape" x1="190" y1="300" x2="190" y2="185" />
+            <line class="FlowShape" x1="190" y1="250" x2="275" y2="250" />
+            <line class="FlowShape" x1="275" y1="250" x2="275" y2="290" />
+            <line class="FlowShape" x1="190" y1="275" x2="220" y2="275" />
+            <line class="FlowShape" x1="220" y1="275" x2="220" y2="298" />
+            <line class="MainShape" x1="210" y1="160" x2="260" y2="150" />
+        </g>
+
+        <g id="leftValve_On">
+            <circle cx="190" cy="210" r="15" />
+            <line x1="190" y1="195" x2="190" y2="225" />
+        </g>
+        <g id="leftValve_Off">
+            <circle cx="190" cy="210" r="15" />
+            <line x1="175" y1="210" x2="205" y2="210" />
+        </g>
+
+        <g id="leftPump1_Off">
+            <rect class="ValvePumpInactive" x="177.5" y="300" width="25" height="25" />
+            <line class="ValvePumpInactive" x1="182.5" y1="312.5" x2="197.5" y2="312.5" />
+        </g>
+        <g id="leftPump1_On">
+            <rect x="177.5" y="300" width="25" height="25" />
+            <line x1="190" y1="300" x2="190" y2="325" />
+        </g>
+
+        <g id="leftPump2_Off">
+            <rect class="ValvePumpInactive" x="207.5" y="300" width="25" height="25" />
+            <line class="ValvePumpInactive" x1="212.5" y1="312.5" x2="227.5" y2="312.5" />
+        </g>
+        <g id="leftPump2_On">
+            <rect x="207.5" y="300" width="25" height="25" />
+            <line x1="220" y1="300" x2="220" y2="325" />
+        </g>
+
+        <!--Right-->
+        <g id="rightPipes">
+            <line class="FlowShape" x1="410" y1="300" x2="410" y2="185" />
+            <line class="FlowShape" x1="410" y1="250" x2="325" y2="250" />
+            <line class="FlowShape" x1="325" y1="250" x2="325" y2="290" />
+            <line class="FlowShape" x1="410" y1="275" x2="380" y2="275" />
+            <line class="FlowShape" x1="380" y1="275" x2="380" y2="298" />
+            <line class="MainShape" x1="390" y1="160" x2="340" y2="150" />
+        </g>
+
+        <g id="rightValve_On">
+            <circle cx="410" cy="210" r="15" />
+            <line x1="410" y1="195" x2="410" y2="225" />
+        </g>
+        <g id="rightValve_Off">
+            <circle cx="410" cy="210" r="15" />
+            <line x1="395" y1="210" x2="425" y2="210" />
+        </g>
+
+        <g id="rightPump1_Off">
+            <rect class="ValvePumpInactive" x="367.5" y="300" width="25" height="25" />
```

Comment:
Hi @ThePilot916 

I think the missing ValvePumpInactive here makes the closed rightPump1 appears in green instead orange

![image](https://user-images.githubusercontent.com/121706/91665514-6bd1fe00-eaf6-11ea-8676-41b6b475950f.png)